examples/multimodal/README.md

```diff
@@ -125,30 +131,30 @@ In another terminal:
 curl http://localhost:8000/v1/chat/completions \
   -H "Content-Type: application/json" \
   -d '{
-    "model": "llava-hf/llava-1.5-7b-hf",
-    "messages": [
-      {
-        "role": "user",
-        "content": [
-          {
-            "type": "text",
-            "text": "What is in this image?"
-          },
-          {
-            "type": "image_url",
-            "image_url": {
-              "url": "http://images.cocodataset.org/test2017/000000155781.jpg"
+      "model": "llava-hf/llava-1.5-7b-hf",
+      "messages": [
+        {
+          "role": "user",
+          "content": [
+            {
+              "type": "text",
+              "text": "What is in this image?"
+            },
+            {
+              "type": "image_url",
+              "image_url": {
+                "url": "http://images.cocodataset.org/test2017/000000155781.jpg"
+              }
             }
-          }
-        ]
-      }
-    ],
-    "max_tokens": 300,
-    "stream": false
-  }'
+          ]
+        }
+      ],
+      "max_tokens": 300,
+      "stream": false
+    }'
 ```
 
 You should see a response similar to this:
-```
+```json
```

Comment:
Good add. Can we add it to the section above as well?

Reply:
done.